target/linux/ath79/nand/base-files/etc/board.d/01_leds

```diff
@@ -10,6 +10,10 @@ glinet,gl-ar300m-nand|\
 glinet,gl-ar300m-nor)
 	ucidef_set_led_netdev "lan" "LAN" "green:lan" "eth0"
 	;;
+netgear,r6100)
+	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
+	ucidef_set_led_switch "wan-amber" "WAN (amber)" "amber:wan" "switch0" "0x1e"
```

Comment:
If it's a WAN LED, we should not hijack it for LAN in default setup...

Reply:
The WAN LED is actually two LEDs - one green, one amber -, it's again the WNDRs which are set up similarly:

```
netgear,wndr4500-v3)
        ucidef_set_led_switch "wan-amber" "WAN (amber)" "amber:wan" "switch0" "0x20"
```

Reply:
But 0x20 matches (only) the WAN LED for the wndr4500-v3, and it has nothing set for green. So, you have one LED named "WAN" that indicates WAN port status there.

In your case, you have two LEDs named WAN of which one shows the WAN port status and the other one shows the LAN ports' status. I'd prefer that for default setup you choose one of the colors and link that to eth1. Some Netgear devices also have a setup where one color is for link status and the other one is for activity; however, I'd make that dependent on what the vendor did.